.github/workflows/download_pipeline.yml

```diff
@@ -28,11 +30,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install Nextflow
-        uses: nf-core/setup-nextflow@v1
+        uses: nf-core/setup-nextflow@v2
+
+      - name: Disk space cleanup
```

Comment:
I do not know where the `jlumbroso/free-disk-space` step comes from. I have not heard of any issues with disk space so far, and it does uninstall a lot of packages that may or may not be needed. Probably no harm done here, but I believe also not strictly necessary.

Reply:
I will keep it in mind in case it causes issues in the future.